README.md

```diff
@@ -48,8 +54,29 @@ When using the `-D`/`--dist_est` ARCS option to estimate gap sizes, the user is
 
 An example bash script on how to run the ARCS+LINKS pipeline can be found at: Examples/pipeline_example.sh
 
-you can test your installation by following instructions at: Examples/arcs_test-demo/README.txt
-and compare your output to the files provided at: Examples/arcs_test-demo/output/ 
+### Running ARCS in default mode
+
+The default mode uses alignments of linked reads to contigs to scaffold the input contigs.
+
+To run the pipeline in default mode, run `Examples/arcs-make arcs`. For more info check `Examples/arcs-make help`.
+
+To run the `arcs` executable in default mode, run `arcs <alignments>`. For descriptions of all arguments, run `arcs --help`.
+
+### Running ARCS in '--arks' mode
+
+To run the pipeline in ARKS mode, run `Examples/arcs-make arks`. For more info check `Examples/arcs-make help`.
+
+To run the `./arcs` executable in ARKS mode, run `./arcs --arks`. For descriptions of all arguments, run `./arcs --help`.
```

Comment:
```suggestion
To run the `arcs` executable in ARKS mode, run `arcs --arks`. For descriptions of all arguments, run `arcs --help`.
```
Forgot to change this one too in my last round of suggestions to keep it consistent with ARCS